node/src/blockchain/blockchain_interface.rs

```diff
@@ -2026,7 +2026,11 @@ mod tests {
 
         assert_eq!(
             actual_error,
-            BlockchainError::QueryFailed(format!("Transport error: Error(Connect, Os {{ code: {}, kind: ConnectionRefused, message: \"Connection refused\" }})", if cfg!(target_os = "macos") { 61 } else { 111 }).to_string())
+            BlockchainError::QueryFailed(format!("Transport error: Error(Connect, Os {{ code: {}, kind: ConnectionRefused, message: \"{}\" }})",
+                                                 if cfg!(target_os = "windows") { 10061 } else if cfg!(target_os = "macos") { 61 } else { 111 },
+                                                 if cfg!(target_os = "windows") { "No connection could be made because the target machine actively refused it." }
+                                                 else { "Connection refused" }).to_string(),
+            )
```

Comment:
By our standard, we'd check out the message probably like this:

    match result {
        Err(BlockchainError::QueryFailed(err_message)) => assert!(
            err_message.contains("Transport error: Error(Connect, Os {{ code:") &&
            err_message.contains("kind: ConnectionRefused"),
            "we got this error msg: {}" but expected specific error fragments, see in the assertion: ,
            err_message
        ),
        Err(e) => panic!("we expected ConnectionRefused error but got this one: {}", e),
        Ok(x) => panic!("we expected an error, but got: {:?}", x),
    };
...to avoid the dynamical, os specific parts.